getpaid/models.py

```diff
@@ -1,5 +1,7 @@
 import sys
 from datetime import datetime
+
+from django.apps import apps
```

Comment:
I'm afraid it would not be the same, as `.apps` is just a conventional module for **declaration** of config class of the application (subclass of `django.apps.AppConfig`) while `django.apps.apps` holds **instances** of config classes of all applications loaded from these `.apps` modules.